tiup/tiup-overview.md

```diff
@@ -65,51 +85,45 @@ Available Commands:
   status      List the status of instantiated components
   clean       Clean the data of instantiated components
   mirror      Manage a repository mirror for TiUP components
-  help        Help about any command or component
-
-Components Manifest:
-  use "tiup list" to fetch the latest components manifest
+  telemetry   Controls things about telemetry
```

Comment:
The language style of this line is different from other lines. See "When used for description, comments should be descriptive rather than imperative" in our [Code Comment Style](https://github.com/pingcap/community/blob/master/contributors/code-comment-style.md#language-for-a-good-comment). But if this is the output of the `tiup --help` command, it's ok to keep it as it is.

Reply:
Yes, it is the output of the `tiup --help` command 😂